app_dart/lib/src/service/scheduler.dart

```diff
@@ -229,6 +229,7 @@ class Scheduler {
     );
     final YamlMap configYaml = loadYaml(configContent) as YamlMap;
     pb.SchedulerConfig schedulerConfig = pb.SchedulerConfig();
+    pb.SchedulerConfig totSchedulerConfig = pb.SchedulerConfig();
```

Comment:
This should get passed in directly as a CiYaml so we're not duplicating code in this method

Reply:
umm sorry for asking for clarification, by "pass directly as CiYaml" are we trying to concatenate smaller pieces of code into a one liner? Maybe my understanding is wrong, I thought that the slug, branch and schedulerConfig have to be initialized beforehand to create a CiYaml class.

Reply:
Thanks for the example! This is very helpful! I have refactored files implementing this topology